src/amp.js

```diff
@@ -63,19 +63,17 @@ try {
 
       maybeValidate(window);
       makeBodyVisible(document, waitForExtensions(window));
-    } catch (e) {
-      makeBodyVisible(document);
```

Comment:
Re: https://github.com/ampproject/amphtml/pull/2146/files#r53512967, we should keep the `catch` block and just re-throw.